bundles/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/types/IntegraType.java

```diff
@@ -79,7 +88,7 @@ public boolean hasExtPayload() {
      * Returns Integra type for given code.
      *
      * @param code
-     *            code to get type for
+     *                 code to get type for
```

Comment:
This looks like a wriong formatting. Can you apply the ESH formatter (should be automatically on save in Eclipse)?

Reply:
Done.